draft-kuhn-quic-bdpframe-extension-01.xml

```diff
@@ -127,6 +127,13 @@
 			BDP_FRAME parameters; 
 			</t>
 			<t>
+			The client can read most of the BDP_FRAME parameters but not 
+			modify them. The sender computes a secured hash with 
+			its own selection of the parameters of the BDP_FRAME,
+			encrypts the hash and send it within the BDP_FRAME.  
+			The client can not read the hash.
```

Comment:
/send/sends/

Reply:
Fixed in https://github.com/NicoKos/resume_connection/commit/dbf99b0ac345108cc95cd1d4b0b5d35b30b4ff71